CMakeLists.txt

```diff
@@ -286,9 +286,9 @@ if(CMAKE_GENERATOR STREQUAL Xcode)
   set_property(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS
     "$(inherited)"
   )
-  # set_property(DIRECTORY APPEND PROPERTY COMPILE_OPTIONS
-  #   "$(inherited)"
-  # )
+  set_property(DIRECTORY APPEND PROPERTY COMPILE_OPTIONS
```

Comment:
Can this be merged with the above block?
```
set_property(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS COMPILE_OPTIONS
     "$(inherited)"
)
```
I.e Does that work?

Reply:
Unfortunately you can specify only one property
https://cmake.org/cmake/help/v3.10/command/set_property.html